src/View/View.php

```diff
@@ -360,12 +360,34 @@ public function __construct(
      * So this method allows you to manipulate them as required after view instance
      * is constructed.
      *
+     * Helpers can be added using {@link addHelper()} method.
+     *
      * @return void
      */
     public function initialize(): void
     {
     }
 
+    /**
+     * Allows adding helpers from initialize() hook, overwriting any previously set ones.
+     *
+     * @param string $helper
+     * @param array<string, mixed> $config
+     * @return void
+     */
+    protected function addHelper(string $helper, array $config = []): void
```

Comment:
This seems like a method we could backport to 4.x reasonably easily and add a deprecation warning to loadHelper. It is also a good candidate for updating automatically with rector.

Reply:
Dont we still need load helper for templates? We just dont need it for setup.

Reply:
Yeah you're right, I wasn't thinking about that.